topic-operator/src/main/java/io/strimzi/operator/topic/Config.java

```diff
@@ -108,6 +108,12 @@ private Value(String key, Type<? extends T> type, boolean required) {
     public static final String TC_TLS_KEYSTORE_PASSWORD = "STRIMZI_KEYSTORE_PASSWORD";
     public static final String TC_TLS_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM = "STRIMZI_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM";
 
+    public static final String TC_STORE_TOPIC = "STRIMZI_STORE_TOPIC";
+    public static final String TC_STORE_NAME = "STRIMZI_STORE_NAME";
+    public static final String TC_APPLICATION_ID = "STRIMZI_APPLICATION_ID";
+    public static final String TC_APPLICATION_SERVER = "STRIMZI_APPLICATION_SERVER";
+    public static final String TC_STALE_RESULT_TIMEOUT = "STRIMZI_STALE_RESULT_TIMEOUT";
```

Comment:
Can we include the unit? 
```suggestion
    public static final String TC_STALE_RESULT_TIMEOUT_MS = "STRIMZI_STALE_RESULT_TIMEOUT_MS";
```

Reply:
Sure.